syft/formats/cyclonedxxml/format.go

```diff
@@ -7,14 +7,66 @@ import (
 	"github.com/anchore/syft/syft/sbom"
 )
 
-const ID sbom.FormatID = "cyclonedx-xml"
+const ID sbom.FormatID = "cyclonedx"
```

Comment:
I think this is a consequential change, can you expound on this a bit? (I think there is an alias for cyclonedx already, which I want to say defaults to json, but I haven't verified this)

Reply:
actually, this doesn't seem right: this is changing the ID for this format to something more generic, but the ID should be as specific as possible. SPDX does this by having a list of IDs, where the first is the ID and the remaining are unique aliases https://github.com/anchore/syft/blob/9de4129638a60dbef0c87b964bdcee561b0ee12a/syft/formats/spdxtagvalue/format.go#L9

Then that list is used in the format object https://github.com/anchore/syft/blob/9de4129638a60dbef0c87b964bdcee561b0ee12a/syft/formats/spdxtagvalue/format.go#L18

This PR seems to be reversing this, where the aliases are more specific 

https://github.com/anchore/syft/pull/2123/files#diff-353e0a54e1d8afae37163dec0733b3e941580dc79824b40b9bf35e649efa28b0R20

Since all of these instances are the same can you make a list and share that similar to SPDX?

Reply:
Ahh - I think this was just an unexpected change on my part thanks for the catch - I looked back at the diff and the original format had the alias of `cyclonedx`. The original ID was `cyclonedx-xml`. I think a unit test was failing and to fix it I put the wrong alias back in the wrong spot. I'll fix this now.

Reply:
@wagoodman this should now be reverted to where there are no ID changes and aliases are consistent across the new formats